dbt/adapters/databricks/python_models/python_submissions.py

```diff
@@ -97,39 +202,20 @@ def _build_job_permissions(self) -> List[Dict[str, Any]]:
             )
             access_control_list.append(acl_grant)
 
-        return access_control_list
-
-    def _build_job_owner(self) -> Tuple[str, str]:
-        """
-        :return: a tuple of the user id and the ACL attribute it came from ie:
-            [user_name|group_name|service_principal_name]
-            For example: `("mateizaharia@databricks.com", "user_name")`
-        """
-        curr_user = self.api_client.curr_user.get_username()
-        is_service_principal = self.api_client.curr_user.is_service_principal(curr_user)
-
-        if is_service_principal:
-            return curr_user, "service_principal_name"
-        else:
-            return curr_user, "user_name"
+        return access_control_list + self.acls
 
-    def _submit_job(self, path: str, cluster_spec: dict) -> str:
-        job_spec: Dict[str, Any] = {
-            "task_key": "inner_notebook",
-            "notebook_task": {
-                "notebook_path": path,
-            },
-        }
-        job_spec.update(cluster_spec)  # updates 'new_cluster' config
 
-        # PYPI packages
-        packages = self.parsed_model["config"].get("packages", [])
+class PythonLibraryConfigurer:
```

Comment:
nit: is this abstract class? Should it inherit from `ABC`

Reply:
Not abstract, just a static method holder.  The real question is whether to move the function out to the module layer, since it doesn't really need a class to hold it (since it has no state).  I'm kind of on the fence about bare functions, but maybe I should use it more often in cases like this.